webpack.config.babel.js

```diff
@@ -58,6 +58,7 @@ module.exports = {
         new webpack.ProvidePlugin({
             [options.jsxPragma]: path.join(__dirname, "utils", "element"),
         }),
+        new webpack.optimize.UglifyJsPlugin({compress: {warnings: false}})
```

Comment:
Oh hmmm, I'm noticing that this is minifying `tota11y.js` too, but we want to keep that uncompressed.

I noticed this by running:
```
rm -rf build/
npm run build
```

and then viewing the contents of `build/tota11y.js`. Is there an easy way we can make this plugin only apply for the production case? (See `prod` in package.json's "scripts" field).